hadoop-hdfs-project/hadoop-hdfs/src/test/java/org/apache/hadoop/hdfs/server/namenode/ha/TestConsistentReadsObserver.java

```diff
@@ -123,7 +123,7 @@ public void testRequeueCall() throws Exception {
         + CommonConfigurationKeys.IPC_BACKOFF_ENABLE, true);
 
     NameNodeAdapter.getRpcServer(nn).refreshCallQueue(configuration);
-    assertEquals(1, NameNodeAdapter.getRpcServer(nn).getTotalRequests());
+    assertTrue(NameNodeAdapter.getRpcServer(nn).getTotalRequests() > 0);
```

Comment:
if you have an assertion where a useful message isn't autogenerated the way assertequals does, afraid you will need to add a message *or* use assertJ to describe the assert. sorry.

Reply:
thanks! it is pretty verbose for the little things, but i find assertj lovely for complex things, especially collections